### PR TITLE
Build operator images for x86-64 and ARM64

### PR DESCRIPTION
**Pull request by skitt**
The index image is amd64-only.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Comment by dfarrell07** [comment]
```
bin/lichen -c .lichen.yaml bin/subctl bin/linux/amd64/submariner-operator
2022/05/03 16:39:09 failed to evaluate licenses: unrecognised line: 	
make: *** [Makefile:155: licensecheck] Error 1
[16:39:09.946] [submariner-operator]$ make --debug=b licensecheck
```

**Comment by skitt** [comment]
> ```
> bin/lichen -c .lichen.yaml bin/subctl bin/linux/amd64/submariner-operator
> 2022/05/03 16:39:09 failed to evaluate licenses: unrecognised line: 	
> make: *** [Makefile:155: licensecheck] Error 1
> [16:39:09.946] [submariner-operator]$ make --debug=b licensecheck
> ```

`lichen` had a few issues with binaries compiled with Go 1.18, fixed over the last couple of months (including https://github.com/uw-labs/lichen/pull/21 by yours truly). Once `go build` picks the latest release, all these errors will be resolved.

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Comment by dfarrell07** [comment]
This is timing out while still running the E2E, I think without any failure. Note that this is different than the timeouts in the post-mortem of failing E2E that should be handled by failing fast.

**Comment by tpantelis** [comment]
> This is timing out while still running the E2E, I think without any failure. Note that this is different than the timeouts in the post-mortem of failing E2E that should be handled by failing fast.

Perhaps building the additional images adds enough time to exceed the 30 min total for E2E?

**Comment by skitt** [comment]
> > This is timing out while still running the E2E, I think without any failure. Note that this is different than the timeouts in the post-mortem of failing E2E that should be handled by failing fast.
> 
> Perhaps building the additional images adds enough time to exceed the 30 min total for E2E?

E2E shouldn’t build the additional images, and I don’t see the doing so (in fact, I forgot to add a GHA to build the multiarch images...). Some of the logs suggest that the builds were insanely slow — it took four minutes to build `subctl`...

**Comment by skitt** [comment]
The UPX logs in the multi-arch job show that this is building amd64 and arm64 binaries and images.

**Comment by dfarrell07** [comment]
We can go ahead and merge this?

File: .github/workflows/multiarch.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Multi-arch builds
```
Comment (dfarrell07) [inline .github/workflows/multiarch.yml:2]:
```suggestion
name: Multi-arch Builds
```

This same nit about the UI applies to all the similar PRs. :sweat_smile:

**Pull request merged**